### ISSUE TITLE
migrate blueprint syntax to remove testem deprecation [good-first-issue]

### ISSUE THREAD

**Issue by efx** [ember-cli/ember-cli#8708]
I noticed the default output for testem configuration uses the old syntax for specifying browser arguments. This results in a deprecation warning logged to whatever file you specify using the `--testem-debug` flag. See [ember-new-output](https://github.com/ember-cli/ember-new-output/blob/master/testem.js#L10-L12) for current output.
# migrated output
```js
  browser_args: {
    Chrome: {
      all: [
        // --no-sandbox is needed when running Chrome inside a container
        process.env.CI ? '--no-sandbox' : null,
        '--headless',
        '--disable-gpu',
        '--disable-dev-shm-usage',
        '--disable-software-rasterizer',
        '--mute-audio',
        '--remote-debugging-port=0',
        '--window-size=1440,900'
      ].filter(Boolean)
    }
  }
```
---

Output from `ember version --verbose && npm --version && yarn --version`:
```
ember-cli: 3.10.1
http_parser: 2.8.0
node: 10.16.0
v8: 6.8.275.32-node.52
uv: 1.28.0
zlib: 1.2.11
brotli: 1.0.7
ares: 1.15.0
modules: 64
nghttp2: 1.34.0
napi: 4
openssl: 1.1.1b
icu: 64.2
unicode: 12.1
cldr: 35.1
tz: 2019a
os: darwin x64
6.9.0
```

**Comment by rwjblue** [comment]
Thanks for reporting! Would love a PR to fix...

**Comment by efx** [comment]
@rwjblue I was incorrect; You had fixed this already in #7560. and I didn't understand ember-cli uses its own blueprint (`./blueprints/app/files/testem.js`). So while ember-new-output's generated files differ, I don't think that is this issue.
I generated a new app using master and it does not warn about the configuration.
